_resources/port1.0/group/crossgdb-1.0.tcl

```diff
@@ -0,0 +1,179 @@
+                            port:libiconv \
+                            port:ncurses \
+                            port:zlib \
+                            port:boehmgc
+
+        post-patch {
+            # Fix the info pages and related stuff.
+            #
+            # path:     path to the doc directory (e.g. gas/doc/)
+            # makefile: path to Makefile.in (e.g. gas/doc/Makefile.in)
+            # name:     name of the info page (e.g. as)
+            # suffix:   suffix of the source page (texinfo or texi)
+            #
+            #   path        makefile                name        suffix
+            set infopages {
+                bfd/doc     bfd/doc/Makefile.in     bfd         texi
+                gdb/doc     gdb/doc/Makefile.in     annotate    texinfo
+                gdb/doc     gdb/doc/Makefile.in     gdb         texinfo
+                gdb/doc     gdb/doc/Makefile.in     stabs       texinfo
+            }
+
+            foreach { path makefile name suffix } ${infopages} {
+                set src      ${worksrcpath}/${path}/${name}.${suffix}
+                set makefile ${worksrcpath}/${makefile}
+                # If the makefile doesn't exists, skip it.
+                if { ! [file exists ${makefile}] } {continue}
+
+                # Fix the source
+                reinplace -q "s|setfilename ${name}.info|setfilename ${crossgdb.target}-${name}.info|g" ${src}
+                reinplace -q "s|(${name})|(${crossgdb.target}-${name})|g" ${src}
+                reinplace -q "s|@file{${name}}|@file{${crossgdb.target}-${name}}|g" ${src}
+
+                # Rename the source
+                file rename ${worksrcpath}/${path}/${name}.${suffix} \
+                    ${worksrcpath}/${path}/${crossgdb.target}-${name}.${suffix}
+
+                # Fix the Makefile
+                reinplace -q -E "s:\[\[:<:\]\]${name}\\.(info|pod|${suffix}):${crossgdb.target}-&:g" ${makefile}
+
+                # Fix install-info's dir.
+                # (note: this may be effectless if there was no info dir to be fixed)
+                reinplace -q "s|--info-dir=\$(DESTDIR)\$(infodir)|--dir-file=\$(DESTDIR)\$(infodir)/${crossgdb.target}-gdb-dir|g" ${makefile}
+            }
+
+            # Do not install libiberty
+            reinplace -q {/^install:/s/ .*//} ${worksrcpath}/libiberty/Makefile.in
+        }
+
+        # Older Apple clang versions segfault, c.f.,
+        # https://trac.macports.org/ticket/56883
+        compiler.blacklist  {clang < 601}
+        compiler.thread_local_storage yes
+
+        # Needs C++11; halfway redundant due to the blacklist above, but make
+        # sure selected compiler supports the standard - getting rid of old
+        # Apple GCC versions and the like?
+        compiler.cxx_standard 2011
+
+        compiler.blacklist  {clang < 601}
+
+        configure.dir       ${workpath}/build
+        configure.cmd       ${worksrcpath}/configure
+        configure.args      \
```

Comment:
no need for a line continuation here, you can just start with the first argument